tests/test_equilibrium.py

```diff
@@ -48,7 +48,7 @@ def test_compute_theta_coords():
 def test_map_coordinates():
     """Test root finding for (rho,theta,zeta) for common use cases."""
     # finding coordinates along a single field line
-    eq = get("W7-X")
+    eq = get("DSHAPE_current")
```

Comment:
A better option would probably be to make the w7x example have a fixed current profile, since that's more realistic. I think the iota profile it has was a fit from a fixed current vmec solve

Reply:
Or use precise_QA which has a fixed current (albeit at zero) profile?